pigpen-cascading/src/main/java/pigpen/cascading/OperationUtil.java

```diff
@@ -13,6 +13,8 @@
 
 public class OperationUtil {
 
+  public static final String SENTINEL_VALUE = "39c63d213f5cba3c7";
```

Comment:
Fair enough. I was also trying to use something that is not too large, but correctness guarantee comes first.